scripts/release_ios_sdk.sh

```
@@ -1,305 +1,533 @@
 
-    # Create release
-    create_release "$NEW_VERSION"
+  local paths_to_add=("$VERSION_FILE")
+  [[ -f "$CHANGELOG_FILE" ]] && paths_to_add+=("$CHANGELOG_FILE")
+  [[ -f "$README_ROOT" ]] && paths_to_add+=("$README_ROOT")
+  [[ -f "$README_SDK" ]] && paths_to_add+=("$README_SDK")
 
-    # Create GitHub release
-    create_github_release "$NEW_VERSION"
+  git add "${paths_to_add[@]}"
+  git commit -m "Release v$new_version
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)